agoradatatools/etl/transform.py

```diff
@@ -341,8 +341,21 @@ def create_proteomics_distribution_data(datasets: dict) -> pd.DataFrame:
     return pd.concat(transformed)
 
 
-def apply_custom_transformations(datasets: dict, dataset_name: str, dataset_obj: dict):
+def transform_biodomains(datasets: dict):
+    biodomains = datasets['biodomains']
+    interesting_columns = ['ensembl_id', 'biodomain', 'goterms']
+    biodomains = biodomains[interesting_columns]
+
+    biodomains = biodomains.groupby(['ensembl_id', 'biodomain'])['goterms'].apply(list).reset_index()
```

Comment:
Can you add a comment explaining what this line of code does?

Reply:
done!